megatron/arguments.py

```diff
@@ -73,6 +73,9 @@ def validate_args(args, defaults={}):
     assert args.world_size % args.tensor_model_parallel_size == 0, 'world size'\
         ' ({}) is not divisible by tensor model parallel size ({})'.format(
             args.world_size, args.tensor_model_parallel_size)
+    # Zero bubble pipeline is defined on deepspeed's scheduler
+    if args.zero_bubble_pipeline:
+        assert args.deepspeed, 'Use DeepSpeed to use zero-bubble'
```

Comment:
Maybe we name this as zbh1 (flag: enable_zbh1_pipeline). Also change the naming of the pipeline to ZBH1 in other files.